Atomic/Export.py

```diff
@@ -0,0 +1,107 @@
+"""
+export docker images, containers and volumes into a filesystem directory.
+"""
+import sys
+import os
+import subprocess
+import Atomic.util as util
+
+def export_docker(graph, export_location):
+    """
+    This is a wrapper function for exporting docker images, containers
+    and volumes.
+    """
+
+    if os.geteuid() != 0:
+        exit("You need to have root privileges to run atomic export."
+             "\nPlease try again, this time using 'sudo'. Exiting.")
+
+    if not os.path.isdir(export_location):
+        os.mkdir(export_location)
```

Comment:
Should be in try block.  use os.makedirs() in case export_location includes directories that do not exist.

os.mkdir("/foobar/does/not/exist") will blow up.